finality-aleph/src/justification.rs

```diff
@@ -232,11 +233,12 @@ where
             Some(authorities) => authorities.to_vec(),
             None => return None,
         };
-
-        Some(KeyBox {
+        // Below we use a fake index 0 of a node -- we never use this keybox for signing so that's OK, but should consider
+        // refactoring this in the future so that verification and signing a separate (not combined within a KeyBox).
+        Some(KeyBox::new(
```

Comment:
I made a small improvement so that the keybox is created twice instead of 3 times overall. The general issue with session_authorities I'm aware of and I think that this deserves a deeper refactor, just not sure if we can afford it now :)